src/style/app.module.css

```diff
@@ -2618,7 +2625,12 @@ input[type='radio']:checked + label:hover {
   color: var(--grey-bg-color-dark);
 }
 
-.topRadius {
+.people_card_header {
+  background-color: var(--grey-light);
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with future attempts to having Dark Mode.
- For example `--search-button-bg` is much better than `--light-blue`